mlrun/datastore/targets.py

```diff
@@ -407,16 +397,8 @@ def __init__(
             time_partitioning_granularity,
             max_events,
             flush_after_seconds,
-            after_state,
             schema=schema,
         )
-        if after_state:
-            warnings.warn(
-                "The after_state parameter is deprecated. Use after_step instead",
-                # TODO: In 0.7.0 do changes in examples & demos In 0.9.0 remove
-                PendingDeprecationWarning,
-            )
-            after_step = after_step or after_state
```

Comment:
This change broke existing functionality, which caused `TestFeatureStore::test_split_graph` to fail.

More generally, when removing the deprecation warning and handling for a parameter, the parameter itself should be deleted as well.

Reply:
That's not the parameter, that's passing of the parameter to the superclass.  The parameter is on line 391. And of course, when removing a parameter, it's necessary to make sure it's not used by any client code within the same repo, and no longer appears in any documentation either.

This is important so that we establish a proper protocol for deprecation in the future.